frontend/www/index.html

```diff
@@ -29,6 +29,16 @@
   <!-- highcharts -->
   <script src="//code.highcharts.com/stock/highstock.js"></script>
   <script src="lib/highcharts-ng/dist/highcharts-ng.js"></script>
+  <script src="lib/highcharts/highcharts.js"></script>
```

Comment:
you don't need highcharts again if highstocks are alreay included. Highstocks includes everything highcharts has and more. 

However, if you are using stuff from highcharts-more, than that is needed.